packages/publicui/routes/index.tsx

```diff
@@ -71,7 +83,9 @@ export default function Index() {
       <body>
         <div role="navigation" aria-label="Toolbar" id="toolbar" />
         <div id="main">
-          <Outlet />
+          <RouterProvider navigate={navigate} useHref={useHref}>
```

Comment:
@ksuess We need it at the top of the Seven app (root.tsx). This used to be like this some months ago, we have to fix it before the sprint. We originally created `@plone/providers` for that, but could be that we do not need them anymore in Seven (we might do in other frameworks).

Do you want to take care of it? or do you want me to contribute to this PR?  let's merge it before the sprint!

Reply:
/cc @pnicolli

Reply:
well, this topic is complex. Give me this day to read and try some things, then I head it over to the people that have  react router 7 experience for more than my three days… ;-)